nim-mode.el

```diff
@@ -134,7 +134,11 @@
   (setq-local electric-indent-inhibit t)
   (setq-local electric-indent-chars '(?: ?\s))
   (when electric-indent-mode
-    (define-key nim-mode-map [remap delete-backward-char] 'nim-electric-backspace)))
+    (define-key nim-mode-map [remap delete-backward-char] 'nim-electric-backspace))
+
+
+  ;; Set `compile-command' to the correct initial value
+  (nim-compile--set-compile-command))
```

Comment:
Please back this function to `nim-compile.el`. I'm thinking to remove `(require 'nim-compile)` from `nim-mode.el` for lazy loading.